plugins/zoom/bin/icon_zoom

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Zoom"
 Vendor = "rapid7"
-Version = "4.1.11"
+Version = "4.1.13"
```

Comment:
if 4.1.12 hasn't been released yet, is it worth just combining these 2 versions into 1 release to avoid a gap in versions deployed to production ?